conjur-oss/templates/deployment.yaml

```diff
@@ -131,6 +131,13 @@ spec:
             secretKeyRef:
               name: {{ .Release.Name }}-conjur-database-url
               key: key
+        {{ if .Values.database.remotePassword }}
+        - name: PGPASSWORD
+          valueFrom:
+            secretKeyRef:
+              name: {{ .Release.Name }}-database-remote-password
```

Comment:
Isn't this suppose to be:
`<helm-release>-remote-database-password`
as mention at the readme?

Reply:
No - since all other parameters that are metioned as <helm-release> in readme appear as {{ .Release.Name }} in helmchart

Reply:
I am talking about `remote-database-password` vs `database-remote-password`

Reply:
database-remote-password is a name of K8S secret - it is not mention in README

Reply:
I'm missing something...
![Screen Shot 2020-10-12 at 13 57 30](https://user-images.githubusercontent.com/50704381/95739128-fc256600-0c92-11eb-8f7f-1741813c2f2f.png)

Reply:
Oh, I see - it should be database-remote-password. 
Fixed